src/employee_management/FullTimeEmployee.java

```diff
@@ -1,11 +1,40 @@
 package employee_management;
 
-public class FullTimeEmployee {
-    public int employeeId;
-    public String name;
-    public double monthlySalary;
 
-    public FullTimeEmployee(int employeeId, String name, double monthlySalary) {
+class FullTimeEmployee {
+    private int employeeId;
+    private String name;
+    private double monthlySalary;
+    
+    
+	public int getEmployeeId() {
+		return employeeId;
+	}
+
+	public void setEmployeeId(int employeeId) {
+		this.employeeId = employeeId;
+	}
+
+	public String getName() {
+		return name;
+	}
+
+	public void setName(String name) {
+		this.name = name;
+	}
+
+	public double getMonthlySalary() {
+		return monthlySalary;
+	}
+
+	public void setMonthlySalary(double monthlySalary) {
+		this.monthlySalary = monthlySalary;
+	}
+
+	
+    
+
+     FullTimeEmployee(int employeeId, String name, double monthlySalary) {
```

Comment:
Constructor always needs to be the first method inside the class to improve readability.